_posts/2024-03-25-whats-cooking.md

```diff
@@ -0,0 +1,218 @@
+
+We use fancy AI to figure out what recipes you might like based on your preferences and basket. In addition to just finding recipes, our AI actually creates new ones!
+
+We'll talk about how we built the extension, using stuff like Next.js for the part you see, Spring Boot for the behind-the-scenes stuff, and Amazon's cloud services to make sure everything runs smoothly. Plus, we'll explain how we set up everything step by step.
+
+We'll also dive into how we tested different AI models to see which one worked best. It was like a taste test for AI! And along the way, we'll share the ups and downs we faced while working on this project.
+
+So, join us as we explore the exciting world of AI in cooking, sharing our journey and discoveries along the way.
+
+# Table of contents
+
+- [Introduction](#introduction)
+- [Table of contents](#table-of-contents)
+- [Our Project (DRAFT - Febe)](#our-project-draft---febe)
+- [Used Technologies](#used-technologies) - [Frontend](#frontend) - [Backend](#backend) - [AWS Services](#aws-services) - [CI/CD Pipeline](#cicd-pipeline) - [AI Models](#ai-models)
+- [Application Architecture (DRAFT - Febe)](#application-architecture-draft---febe)
+- [Leveraging AI (DRAFT - Jonathan)](#leveraging-ai-draft---jonathan)
+- [AI Benchmarking](#ai-benchmarking)
+  - [Benchmarking comparison between models](#benchmarking-comparison-between-models)
+  - [Benchmarking OpenAi ChatGPT](#benchmarking-openai-chatgpt)
+  - [Benchmarking Bedrock Titan](#benchmarking-bedrock-titan)
+  - [Benchmarking Cohere Command](#benchmarking-cohere-command)
+  - [Benchmarking Meta Llama2](#benchmarking-meta-llama2)
+- [Conclusion (DRAFT - Jonathan)](#conclusion-draft---jonathan)
+- [Our experience](#our-experience)
+
+# Our Project (DRAFT - Febe)
+
+For our internship project, we dished out something delightful: "What's Cooking with ChatGPT."
+
+Imagine a Chrome extension that serves up AI-crafted recipes based on your online shopping list.
+
+To cook up this magic, we crafted custom web scrapers to gather data from major Belgian online retailers like Colruyt, Delhaize, Albert Heijn, and Aldi.
+
+In the backend (our Java setup), we seamlessly integrated Spring AI, opening the door to a variety of AI models, including heavy hitters like OpenAI's ChatGPT, Command by Cohere, Meta's LLama2 and Amazon's Titan.
+
+On the frontend, we spruced up the user interface with React and Next.js. Initially simple, just spitting out recipes, but soon we added a pinch of customization, letting users fine-tune preferences like dietary restrictions and culinary whims.
+
+But hold onto your chef's hat! We also added a nifty feature that conjures up a snapshot of what your dish might look like. And, of course, you can save those recipe gems for later in a convenient text file.
+
+# Used Technologies
```

Comment:
Gezien dat dit vooral qua content een opsomming is zou ik dit onderwerp als laatste zetten.